### DISCUSSION TITLE
Change Log Date and Time

### DISCUSSION THREAD

**Discussion by thaubein** [netbox-community/netbox#15476] [General]
I recently noticed that change log time is off by a couple of hours. The OS Time and Date are correct. Where does NetBox maintain the time and date for the change log.?

**Comment by thaubein** [comment]
Current version: v3.6.1

**Comment by AADIP** [comment]
It's in the configuration.py file under the `TIME_ZONE` variable.

**Comment by thaubein** [comment]
Thank you.!
